packages/adapter-cloudflare-workers/README.md

```diff
@@ -14,16 +16,55 @@ Generate this file using `wrangler` from your project directory
 $ wrangler init --site my-site-name
 ```
 
-Then configure your sites build directory in the config file:
+Add the adapter to your `svelte.config.js`:
+
+```js
+// Import the adapter
+import adapter from '@sveltejs/adapter-cloudflare-workers';
+// Here the config-block
+const config = {
+    kit: {
+        // hydrate the <div id="svelte"> element in src/app.html
+        target: '#svelte',
+        adapter: adapter()
+    }
+};
+...
+```
+
+Now you should get some details from Cloudflare. You should get your:
+
+1. Account ID
+2. And your Zone-ID (Optional)
+
+Get them by visiting your Cloudflare-Dashboard and click on any domain. There, you can scroll down and on the left, you can see your details under **API**.
+
+Then configure your sites build directory and your account-details in the config file:
 
 ```toml
-[site]
-bucket = "./build"
-entry-point = "./workers-site"
+account_id = 'YOUR ACCOUNT_ID'
+zone_id    = 'YOUR ZONE_ID' # optional, if you don't specify this a workers.dev subdomain will be used.
+site = {bucket = "./build", entry-point = "./workers-site"}
 ```
 
 It's recommended that you add the `build` and `workers-site` folders (or whichever other folders you specify) to your `.gitignore`.
 
+Now, log in with wrangler:
+
+```sh
+$ wrangler login
+```
+
+Build your project and publish it:
+
+```sh
+$ npm run build && wrangler publish
+```
+
+**You are done!**
+
+Credits to [Scien.cx](https://www.scien.cx/2021/06/21/deploying-sveltekit-using-cloudflare-workers/)
```

Comment:
I'm a bit hesitant to link to outside blog posts because we have no idea if they will be kept up-to-date